src/fastoad/notebooks/02_explanation/04_OpenMDAO.ipynb

```diff
@@ -0,0 +1,1010 @@
+{
```

Comment:
<blockquote>Here we can see that both our problem<code>s</code> converged on the same value.</blockquote><p> </p><blockquote>We get a very similar result to what we obtained in pure Python. As a reminder, we found, for the same input parameters, a <strong>MTOW</strong> of around 1065.9 kg.</blockquote><p>With my suggestion about lowering the tolerance, you can write:</p><p>"We get an identical result to what we obtained in pure Python."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Changed in dfe233f7, also put the right result now